docs/api/v2.yaml

```diff
@@ -83,6 +83,37 @@ paths:
           $ref: '#/components/responses/NotFoundError'
         '500':
           $ref: '#/components/responses/ServerError'
+  /reconciliation-report/{date}:
+    get:
+      summary: fetches a list of docket entries served on a date
+      operationId: 
+      tags:
+        - irs
+      description: |
+        By passing in a date, you can retrieve a list of docket entries served on that date
+      parameters:
+        - in: path
+          name: date
+          description: A specific date, Eastern time with format YYYY-MM-DD; or 'today'
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: 
```

Comment:
Just flagging so it doesn't get lost, but I this needs a string value to pass validation. I'm pretty sure @kkoskelin has a fix incoming.